dromedary.py

```diff
@@ -60,6 +69,14 @@
 `OPAM_SWITCH_ENV_CMD` containing the name of the current or given opam
 switch."""
 
+DEFAULT_OPAM_SWITCH_NAME = "./"
```

Comment:
Reading the rest of the code and coming back here: this feels strange as a "name". It looks like you might end up calling `opam switch create` with the name set as `"./"` which doesn't seem like a good idea. Can you explain a bit more what would happen (I might be wrong), or update this logic in a more clear way (you are allowed to choose a default name like `generated-switch-for-buck2`, or whatever)

Reply:
The problem is the naming, Opam calls this `plain name` or `directory name` or just `name`. This argument to `opam switch create` can be a directory or a name. If it is a directory (relative or absolute), a "sandbox" is created in this directory, if it is a name, a global switch with that name is created (in `~/.opam/`) but only, if a compiler is given too or `name` is the "name" of a compiler. What exactly the "name" of a compiler is also a bit complicated ;)
The idea is that the default is creating a sandbox in the directory the config file is located in, as the "normal" usage is `opam switch create ./ ocaml-base-compiler` in the project's root. 

I did think about naming it `name_or_path`, but I decided to stay "close" to Opam's usage.

Reply:
👍